sphinxcontrib/ocaml/autodocumenters.py

```diff
@@ -156,19 +161,17 @@ class ModuleDocumenter(sphinx.ext.autodoc.Documenter):
 
     def generate(self, more_content=None, real_modname=None, check_module=False, all_members=False):
         if self.env.config.ocaml_source_directories is None:
-            # @todo Use Sphinx's warnings and errors infrastructure
-            print("ERROR: please set ocaml_source_directories in conf.py")
-            exit(1)
+            raise SphinxError('Please set ocaml_source_directories in conf.py')
 
         module_name = self.directive.arguments[0]
         for d in self.env.config.ocaml_source_directories:
-            interface_file_name = "{}/{}.mli".format(d, module_name)
-            if os.path.isfile(interface_file_name):
+            file_name = [f for f in os.listdir(d) if f.lower() == "{}.mli".format(module_name.lower())][0]
```

Comment:
Agreed, I was using this tool on my projects where I match the case of file names and modules, but this doesn't seem to be a widespread convention in the OCaml community.